src/sdg/organisaties/management/commands/send_notification_mail.py

```diff
@@ -0,0 +1,104 @@
+import datetime
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.core.management import BaseCommand
+from django.db.models import Q
+from django.utils.html import strip_tags
+from django.utils.timezone import now
+
+from compat import render_to_string
+from dateutil.relativedelta import relativedelta
+
+from sdg.accounts.models import User
+from sdg.conf.utils import org_type_cfg
+from sdg.core.models.config import SiteConfiguration
+from sdg.producten.models.product import ProductVersie
+
+
+class Command(BaseCommand):
+    help = f"Send product update notification emails from last ({settings.SDG_MAIL_TEXT_CHANGES_EVERY_DAYS}) days to subscribed users."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--user",
+            metavar=["email", "username"],
+            help="Send to email to one specific user instead based on 'email' or 'username'.",
+        )
+
+    def get_user_by_email(self, user):
+        return (
+            User.objects.filter(email=user, roles__ontvangt_mail=True)
+            .order_by("email")
+            .distinct("email")
+        )
+
+    def get_user_by_username(self, user):
+        user_qs = (
+            User.objects.filter(roles__ontvangt_mail=True)
+            .order_by("email")
+            .distinct("email")
+        )
+        if user_qs:
+            for search_term in user.split():
+                user_qs = user_qs.filter(
+                    Q(first_name__iexact=search_term) | Q(last_name__iexact=search_term)
+                )
+        return user_qs
```

Comment:
Sorry, I thought there was a username but I see there is not. Just by email is fine so you can remove the user first/last name search.